static/scripts/rewards/gift-cards/reveal/redeem-code-html.ts

```diff
@@ -4,31 +4,41 @@ const html = String.raw;
 
 export function getRedeemCodeHtml(transaction: OrderTransaction) {
   return html`
-    <div class="gift-card redeem-code" data-transactionId="${transaction.transactionId}">
-      <h3>Your redeem code</h3>
+    <div class="card-section" id="redeem-code" data-transaction-id="${transaction.transactionId}">
+      <h3>Redeem code</h3>
       <p>xxxxxxxxxxxx</p>
       <p>xxxxxxxxxxxx</p>
       <p>xxxxxxxxxxxx</p>
```

Comment:
These `xxx` seem strange. What are they for?

Reply:
Just a placeholder before the actual redeem codes are displayed.  Similar to how password fields hide the actual password unless you click the reveal button. You can see it in action in the QA video above at 0:55. 

The permit URL where a virtual card has been purchased, if anyone views that page, they see this "xxx". It can be revealed only by the wallet signature of original owner.